lib/results.js

```diff
@@ -3,6 +3,7 @@ var inherits = require('inherits');
 var through = require('through');
 var resumer = require('resumer');
 var inspect = require('object-inspect');
+var has = Object.prototype.hasOwnProperty;
```

Comment:
If `tape` targets ES5 (rather than ES3), or, if we want to add `function-bind` as a dep, then this could be `var has = Function.call.bind(Object.prototype.hasOwnProperty);` or `var has = require('function-bind')(Function.call, Object.prototype.hasOwnProperty);` and the function at the bottom could be removed entirely.